extension/src/stats_agg.rs

```diff
@@ -1018,6 +1089,14 @@ mod tests {
             check_agg_equivalence(&state, &client, &pg2d_agg("covar_pop"), &tk2d_agg_arg("covariance", "population"), EPS1);
             check_agg_equivalence(&state, &client, &pg2d_agg("covar_samp"), &tk2d_agg_arg("covariance", "sample"), EPS1);
 
+            // Skewness and kurtosis don't have aggregate functions in postgres, but we can compute them
+            check_agg_equivalence(&state, &client, &pg_moment_query(3, "test_x"), &tk1d_agg("skewness"), TRILLIONTH);
```

Comment:
I'm wondering why this wouldn't be EPS^3 & EPS^4  (approximately?)

Reply:
Since EPS is (significantly) less than 1, EPS^3 ends up being a significantly tighter bound than EPS, which is the opposite of what we want here.

However, the kurtosis and skewness calculations are also the first case where we actually start seeing significant error creeping up.  While the absolute error is still very tiny, due to the random nature of the test we frequently end up looking at this tiny error relative to a close to zero value.  Given that even here we're seelng a relative error of less than 1e-12, I'm not inclined to actually worry, but I do wonder if we should be using something than relative_eq here.